api/api_delete.py

```diff
@@ -9,5 +9,58 @@
 
 api_delete = Blueprint('api_delete', __name__)
 
+"""
+@api_delete.route('api/songs')
+def delete_song(song)
+    return _handle_request() #sta treba vracati?
 
-# Routes go here
+def _handle_request(wanted_params: str, endpoint: str):
+    logging.info(f"User requested delete of {endpoint}.")
+    try:
+        if invalid_parameters(wanted_params):
+            logging.info("Bad parameter added to request.")
+            return make_response(f"Bad parameter", 420)
+
+        if invalid_param_value():
```

Comment:
invalid_param_value() is called with one argument in its definition, here it has none.